docs/extensions/readme.md

```diff
@@ -1089,7 +1089,10 @@ See [Azure RPC Spec](https://github.com/Azure/azure-resource-manager-rpc/blob/ma
 **Schema**:
 Field Name | Type | Description
 ---|:---:|---
-final-state-via | `string` - one of `azure-async-operation` or `location` or `original-uri` or `operation-location` | `final-state-via` SHOULD BE one of
+final-state-via | `string` - one of `azure-async-operation` or `location` or `original-uri` or `operation-location` | see below
+final-state-schema | A `ref` to the schema of the final result | For languages that make this the result of the LRO
```

Comment:
Thanks for looping me in, @mikekistler. This seems tangential to what I was actually asking about, though: while I like it, I was actually hoping that Cadl could, by default (or as often as possible), emit `final-state-via` (or `final-state-schema` as seen here) - or equivalent in DPG - so that generators know *definitively* what the final schema would be so they can generate method signatures accordingly.

This came up recently when we realized .NET's final state algorithm was wrong (and some test assertions along with it): https://github.com/Azure/autorest.csharp/pull/2728. Looking across other languages, we found they were all a little different. While I asked @lmazuel to consider normalizing this while leading the DPG efforts, I can't help but wonder if Cadl could emit more by default to not require "guesswork" like this to determine final state.

/cc @bterlson